nautobot/users/migrations/0008_make_object_permission_a_changelogged_model.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.23 on 2024-01-02 19:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("users", "0007_alter_objectpermission_object_types"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="objectpermission",
+            name="created",
+            field=models.DateTimeField(auto_now_add=True, null=True),
+        ),
+        migrations.AddField(
+            model_name="objectpermission",
+            name="last_updated",
+            field=models.DateTimeField(auto_now=True, null=True),
```

Comment:
For existing ObjectPermissions, I'm presuming these fields are given a null value on migration automatically, so we don't need a data migration? Just confirming.

Reply:
These fields are actually auto-populated with the timestamp when I ran the migration.
```
# Django version 3.2.23
# Nautobot version 2.1.1b1
# Example Nautobot App version 1.0.0
Python 3.10.13 (main, Dec 19 2023, 03:47:28) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> obj = ObjectPermission.objects.first()
>>> obj.last_updated
datetime.datetime(2024, 1, 2, 22, 7, 48, 950988, tzinfo=<UTC>)
>>> obj.created
datetime.datetime(2024, 1, 2, 22, 7, 1, 489508, tzinfo=<UTC>)
>>> obj = ObjectPermission.objects.last()
>>> obj.last_updated
datetime.datetime(2024, 1, 2, 22, 7, 1, 552173, tzinfo=<UTC>)
>>> obj.created
datetime.datetime(2024, 1, 2, 22, 7, 1, 489508, tzinfo=<UTC>)
>>> 
```